examples/OddJobsCliExample.lhs

```diff
@@ -109,7 +109,7 @@ main = do
           -- `OddJobs.ConfigBuilder`. If you want to actually use
           -- structured-logging you'll need to define your own logging function.
           let jobLogger = defaultTimedLogger logger (defaultLogStr defaultJobType)
-              cfg = mkConfig jobLogger "jobs" dbPool (MaxConcurrentJobs 50) myJobRunner Prelude.id
+              cfg = mkConfig jobLogger "jobs_test" dbPool (MaxConcurrentJobs 50) myJobRunner Prelude.id
```

Comment:
Are the changes to this file for your own testing?

Reply:
In step 1 and project docs, the job table is named `jobs_test`. So its most likely a typo here and this change is required OddJobsCliExample exe to work.

```
ghci> import Database.PostgreSQL.Simple (connectPostgreSQL)
ghci> import OddJobs.Migrations
ghci> conn <- connectPostgreSQL "dbname=jobs_test user=jobs_test password=jobs_test host=localhost"
ghci> createJobTable conn "jobs_test"
```